### PR TITLE
#167723569: separating server contents from UI contents

### PR DESCRIPTION
**Pull request by sntakirutimana72**
#### What does this PR do?
- create UI and server folders in order to separate server contents from UI contents and also rename links inside html pages to match new changes.

#### Description of Task to be completed?
the root directory should look like this
- UI
    - pages
         - admin
    -  css
    -  js
    - images
- server
    - tests
    - routes
    - controllers
    - server.js
- package.json
- package-lock.json
- README.md

#### How should this be manually tested?
- visit my repo and navigating to this very branch

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/167723569

#### Screenshots (if appropriate)
restructure-1

restructure-2

restructure-3

#### Questions:
- Should the package.json files be located inside server folder or outside?

### CODE REVIEW

**Comment by oesukam** [comment]
@sntakirutimana72 Is it `README.dm` or `README.md` under your completed tasks?

File: UI/css/css_home_core.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  }
+}
+
+.button {
+  background: none;
+}
+
+a {
+  text-decoration: none;
+}
```
Comment (oesukam) [inline UI/css/css_home_core.css]: @sntakirutimana72 Blank line required at the end of the file

File: UI/css/css_services_core.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+[s-col] [s-item] {
+  margin: 10px 0;
+  padding: 5px 20px;
+  border-radius: 10px;
+  border-left: 3px solid rgb(125, 209, 177);
+  border-right: 3px solid rgb(125, 209, 177);
+  background-color: rgb(241, 240, 235);
+}
```
Comment (oesukam) [inline UI/css/css_services_core.css]: @sntakirutimana72 Blank line required at the end of the file

File: UI/css/css_signin_core.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  margin: 10% 30%;
+}
+
+.in-out .button {
+  width: 44.8%;
+}
+
+a {
+  text-decoration: none;
+}
```
Comment (oesukam) [inline UI/css/css_signin_core.css]: @sntakirutimana72 Blank line required at the end of the file

File: UI/css/css_signup_core.css [hunk 4]
```diff
@@ -13,19 +13,19 @@
...
+  width: 100%;
+  height: 20px;
+  box-sizing: border-box;
+  background: transparent;
 }
 
 .textInput[birth-date] input, .textInput[gender] input {
-        border: none;
+  border: none;
 }
```
Comment (oesukam) [inline UI/css/css_signup_core.css]: @sntakirutimana72 Blank line required

File: UI/css/css_success_core.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+[success] {
+  height: 100%;
+  padding: 8% 0;
+  min-height: 200px;
+}
+
+[m-menu-bar] {
+  background-color: transparent;
+}
```
Comment (oesukam) [inline UI/css/css_success_core.css]: @sntakirutimana72 Blank line required

File: UI/css/css_trip_form_core.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+@keyframes crea {
+  from {
+    top: 10px;
+    opacity: 0;
+  }
+  to {
+    top: 70px;
+    opacity: 1;
+  }
+}
```
Comment (oesukam) [inline UI/css/css_trip_form_core.css]: @sntakirutimana72 Blank line required

File: UI/css/css_user_core.css [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+  position: absolute;
+  font-size: 20px;
+  right: 5px;
+  top: 0;
+}
+
+[a-book-item] [item-col] span[item-col-header] {
+  padding-bottom: 20px;
+  color: rgb(205, 43, 226);
+}
```
Comment (oesukam) [inline UI/css/css_user_core.css]: @sntakirutimana72 Blank line required

File: UI/css/css_user_forgot_core.css [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+  padding-left: 50px;
+}
+
+input[type='date'] {
+  border: none;
+}
+
+input[type='radio'] {
+  margin-right: 5px;
+}
```
Comment (oesukam) [inline UI/css/css_user_forgot_core.css]: @sntakirutimana72 Blank line required

File: UI/css/css_user_profile_core.css [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+  background: transparent;
+}
+
+[c-val] {
+  padding-left: 50px;
+}
+
+[userId-value][c-val], [creatOn-value][c-val] {
+  padding-left: 8px;
+}
```
Comment (oesukam) [inline UI/css/css_user_profile_core.css]: @sntakirutimana72 Blank line required

**Comment by oesukam** [comment]
@sntakirutimana72 Can you also add screenshots for UI pages?

File: UI/css/css_services_core.css (hunk 2 above)
Comment (sntakirutimana72) [reply UI/css/css_services_core.css]: I'm adding it to every file

**Comment by oesukam** [comment]
@sntakirutimana72 One last change, the `package.json` should be in the *root folder* and not under *server*

File: UI/css/css_services_core.css (hunk 2 above)
Comment (oesukam) [reply UI/css/css_services_core.css]: Yes

**Pull request merged**